application/requirements.txt

```diff
@@ -12,7 +12,7 @@ httpx-oauth
 kubernetes
 kubernetes_asyncio==22.6.5 #  Version locked because of issue https://github.com/kubernetes-client/python/issues/1718 and https://github.com/kubernetes-client/python/issues/1751
 mergedeep
-procrastinate
+procrastinate==0.33.0
```

Comment:
Looks like you didn't merged my changes with requirements versions locking.

Reply:
That's right, I'll pull it after this is merged.